tests/test_collinear_endpoint_cases.py

```diff
@@ -0,0 +1,46 @@
+def test_3():
+    # Test lots of collinear endpoint cases.
+    import random, math, pytest
+    from mt2 import mt2
+    random.seed(0) # If the test fails, we want to be able to repeat the test!
+    for i in range(10000):
+        m_visA = random.uniform(0,10)
+        m_visB = random.uniform(0,10)
+        m_invisA = random.uniform(0,10)
+        m_invisB = random.uniform(0,10)
+        m_parent = max(m_visA+m_invisA, m_visB+m_invisB) + random.uniform(0.1,10) # 0.1 prevents possibility of division by zero at MOOO
+        p_parentA = random.uniform(0,10)
+        p_parentB = random.uniform(0,10)
+        E_parentA = math.sqrt(p_parentA**2 + m_parent**2)
+        E_parentB = math.sqrt(p_parentB**2 + m_parent**2)
+        beta_A = p_parentA/E_parentA  # MOO
+        beta_B = p_parentB/E_parentB  # MOO
```

Comment:
Extraneous lowing

Reply:
No, these are line-refa which link back to the earlier comment about protection from div by zero.

Reply:
(though I had miss spelled the link as MOOO rather than MOO on the referring line, now fixed)